protocols/plate_copying/plate_copying.ot2.py

```diff
@@ -50,7 +52,7 @@ def run_custom_protocol(
         labware.load(destination_container, slotName)
         for slotName in dest_slots]
 
-    source_plate = labware.load(source_container, 'C2')
```

Comment:
C2 is automatically set to 6, and there was already a destination container in slot 6